book/from_flat_php_to_symfony2.rst

```diff
@@ -264,11 +265,9 @@ an individual blog result based on a given id::
     function get_post_by_id($id)
     {
         $link = open_database_connection();
-
         $id = intval($id);
-        $query = 'SELECT created_at, title, body FROM post WHERE id = '.$id;
-        $result = mysql_query($query);
-        $row = mysql_fetch_assoc($result);
+      	$result = $link->query('SELECT created_at, title, body FROM post WHERE id = '.$id);    
```

Comment:
This line looks wrongly indented. It should contain one additional white space. Same for the below line.